pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
Just out of curiosity, are you aware of anyone me trying to run the systemml tests on an "older laptop"?